modules/core/src/test/scala/org/scalasteward/core/MockState.scala

```diff
@@ -18,4 +20,21 @@ object MockState {
 
   def empty: MockState =
     MockState(Vector.empty, Vector.empty)
+
+  // Unable to make this implicit. Ambiguous implicit error
```

Comment:
I am unable to make this implicit. If @fthomas you have any suggestions, let me know. If not, I may have to look into this later with a fresh mind.

Reply:
I think it would be easier to change `MockEnv` to something that is already a `MonadThrowable`. But we should do this later. For now I'm fine with your current solution.

Reply:
#274 changes `MockEnv[A]` from `State[MockState, A]` to `StateT[IO, MockState, A]` so that we get the `MonadError` instance for free and this explicit instance isn't necessary anymore.